core/src/main/java/com/linecorp/armeria/client/retry/RetryRequestStrategy.java

```diff
@@ -43,12 +48,18 @@ public boolean shouldRetry(I request, Throwable throwable) {
     }
 
     /**
-     * Returns whether a request should be retried according to the given request and reponse.
+     * Returns {@link CompletableFuture} that contains a {@link Boolean} flag which indicates that
+     * {@link RetryingClient} needs to retry or not.
+     * <p>If an {@link Exception} happens While processing {@link Request} and {@link Response}, this method
+     * should not set the {@link CompletableFuture} with the {@link Exception}. The {@link Exception}
+     * need to be dealt in the {@link RetryRequestStrategy#shouldRetry(Request, Throwable)} method, so
+     * just set with {@code false}.
```

Comment:
so just set with... -> and the `{@link CompletableFuture}` has to be completed with {@code false}.`